src/System.Drawing.Design.Primitives/src/System/Drawing/Design/IToolboxService.cs

```diff
@@ -0,0 +1,212 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.ComponentModel.Design;
+using System.Runtime.InteropServices;
+namespace System.Drawing.Design
+{
+    /// <summary>
+    /// Provides access to the toolbox in the development environment.
+    /// </summary>
+    [ComImport(), Guid("4BACD258-DE64-4048-BC4E-FEDBEF9ACB76"), InterfaceTypeAttribute(ComInterfaceType.InterfaceIsIUnknown)]
+    public interface IToolboxService
+    {
+        /// <summary>
+        /// Gets the names of all the tool categories currently on the toolbox.
+        /// </summary>
+        CategoryNameCollection CategoryNames { get; }
+
+        /// <summary>
+        /// Gets the name of the currently selected tool category from the toolbox.
+        /// </summary>
+        string SelectedCategory { get; set; }
+
+        /// <summary>
+        /// Adds a new toolbox item creator.
+        /// </summary>
+        /// <param name="creator">creater call back</param>
+        /// <param name="format">format</param>
+        void AddCreator(ToolboxItemCreatorCallback creator, string format);
+
+        /// <summary>
+        /// Adds a new toolbox item creator.
+        /// </summary>
+        /// <param name="creator">creater call back</param>
+        /// <param name="format">format</param>
+        /// <param name="host">host</param>
```

Comment:
Is this what the XML comments were in the codebase from which this was ported?  If so, ok... it's just the param tags aren't particularly helpful :)